internal/services/aws_reservation_service.go

```diff
@@ -139,7 +163,27 @@ func CreateAWSReservation(w http.ResponseWriter, r *http.Request) {
 		logger.Debug().Msgf("Found existing pubkey resource %d, upload not enqueued", pkr.ID)
 	}
 
+	logger.Debug().Msgf("Enqueuing launch instance job for pubkey %d and source %d", pk.ID, reservation.SourceID)
+
+	argsLaunch := &jobs.LaunchInstanceAWSTaskArgs{
+		AccountID:     accountId,
+		ReservationID: reservation.ID,
+		PubkeyID:      pk.ID,
+		ImageID:       reservation.ImageID,
```

Comment:
Image ID for now is AMI so the API needs AMI too. We can add resolving later I guess.

Reply:
here we only need the AMI, so I thought of changing from ImageID to AMI